stacks-signer/src/client/stacks_client.rs

```diff
@@ -358,6 +364,65 @@ impl StacksClient {
         self.get_account_entry(address).map(|entry| entry.nonce)
     }
 
+    /// Get information about the tenures between `chosen_parent` and `last_sortition`
+    pub fn get_tenure_forking_info(
```

Comment:
Mind adding some tests for these two functions?

Reply:
I'm not sure there's much benefit from unit tests for these new `StacksClient` functions. The functions themselves are very light on logic. They use the serde deserialization and serialization methods provided by the structs themselves, and what logic is there is all integration logic (i.e., the handling of the tenure forking info responses depends on how the endpoint responds), so is ultimately best tested with the integration testing anyways.